### ISSUE TITLE
scanFaces options not working

### ISSUE THREAD

**Issue by yiluzhang** [Tarikjrh/vision-camera-face-detector-v3#1]
```jsx
import { FaceDetectionOptions, scanFaces } from 'vision-camera-face-detector-v3';

const scanFaceOption: FaceDetectionOptions = {
  performanceMode: 'accurate',
  landmarkMode: 'all',
  contourMode: 'all',
  classificationMode: 'all',
};
const scannedFaces = scanFaces(frame, scanFaceOption);

<Camera isActive device={device} style={styles.camera} pixelFormat="yuv" frameProcessor={frameProcessor} />
```

scannedFaces: 
```jsx
{"0": {"bounds": {"aspectRatio": 0.8416666666666667, "boundingCenterX": 540.5, "boundingCenterY": 1030.5, "height": 909, "width": 909, "x": 86, "y": 576}, "pitchAngle": -14.236714363098145, "rollAngle": 0.12543772161006927, "yawAngle": 2.460653066635132}}
```

package.json: 
```jsx
"react-native": "0.72.4",
"react-native-vision-camera": "^3.6.11",
"vision-camera-face-detector-v3": "^1.1.2",
```

**Comment by Tarikjrh** [comment]
options are now fixed for ios on the latest version, so please update.
as for android i will look into it on the weekend.

**Comment by Tarikjrh** [comment]
android is now fixed on the latest version (v1.1.4).
